pkg/scanners/template/scanner_template.go

```diff
@@ -0,0 +1,135 @@
+package main
```

Comment:
This needs to be something other than `main`, since it's a library and not a binary.